src/main/java/org/zendesk/client/v2/Zendesk.java

```diff
@@ -722,6 +722,19 @@ public void deleteUser(long id) {
         complete(submit(req("DELETE", tmpl("/users/{id}.json").set("id", id)), handleStatus()));
     }
 
+    public User suspendUser(User user, boolean action) {
```

Comment:
I'm fine with either solution, let me know if you want me to change it (looks like you might still be undecided).

Try reading the Zendesk doc, they don't ever use the word "unsuspend", or "reinstate" or other antonym for "suspend". Tough decisions in trying to make a simple thing simple. :)